main.cpp

```diff
@@ -0,0 +1,92 @@
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class rs_m
+{
+public:
+    vector<double> r; 
+
+    void setResort(vector<double> r)
+    {
+        this->r = r;
+    }
+
+    //getting the size of the resort
+    double GetResortSize() 
+    {
+        int len = r.size();
+        return len;
+    }
+
+    int show()
+    {
+        for (int i = 0; i < r.size(); i++)
+        {
+            cout << r[i] << " ";
+        }
+        cout << endl;
+    }
+};
+
+class RSMclient

```

Comment:
Personally, it took me a while to understand what this class houses and for what purpose, so I'd suggest to rename it a bit. I assume this class is for housing Resort Guest related data and methods, right? If so, would you agree that naming this class `Guest` would be better? I mainly advise this, since we are working with quite a lot of networking code, so naming everything `Client` can get confusing!